### ISSUE TITLE
Pipeline include fails to search `share` directory.

### ISSUE THREAD

**Issue by russelldj** [Kitware/kwiver#1254]
This is an issue induced by #1247 which breaks apart the pipeline for burnout. The example includes the processing pipeline `pipelines/detect_burnin.pipe`, which should be discoverable in `<build dir>/share/kwiver/1.5.1/config` but it fails with the following error. 
`The file does not exist: pipelines/detect_burnin.pipe included from <build dir>/examples/pipelines/burnout_on_video.pipe:1 could not be found in search path`

**Comment by mleotta** [comment]
It looks like somewhere around [here](https://github.com/Kitware/kwiver/blob/master/sprokit/src/applets/pipeline_runner.cxx#L86) we might need to add this:

```c++
const std::string prefix = kwiver::vital::get_executable_path() + "/..";
builder.add_search_path( vital::kwiver_config_file_paths( prefix ) );
```

can you try this @russelldj ?

**Comment by russelldj** [comment]
Works, thanks. I'll put up a PR. Looks like the example video isn't installed either, so I'll look into that.

**Comment by russelldj** [comment]
Fixed by #1258.
